digit-health-services/libraries/services-common/pom.xml

```diff
@@ -0,0 +1,74 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>org.digit.common</groupId>
+    <artifactId>services-common</artifactId>
+    <version>1.0-SNAPSHOT</version>
+    <name>services-common</name>
```

Comment:
health-services-common

Reply:
updated artifactId, name to health-services-common. Updated version to 1.0.0 and added packaging tag.